runner/src/load_generator/gen.rs

```diff
@@ -0,0 +1,251 @@
+use std::collections::HashMap;
+use std::path::PathBuf;
+
+use crate::load_generator::utils::{
+    CeramicConfig, CeramicDidType, CeramicScenarioParameters, StableLoadUser,
+};
+use crate::utils::parse_peers_info;
+use crate::CommandResult;
+use anyhow::Result;
+use ceramic_core::StreamId;
+use clap::Args;
+use keramik_common::peer_info::Peer;
+use tokio::time::{Duration, Instant};
+
+// TODO : Use this to envoke a particular scenario, currently we only have one
+// so this is unused
+#[allow(dead_code)]
+#[derive(Clone, Debug)]
+pub enum WeekLongSimulationScenarios {
```

Comment:
What about a different term instead of `WeekLong` since the run time is configurable? Maybe `LoadGenScenarios`? If so, can replace that throughout this file.